examples/creating_printing_table/creating_printing_table.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015 Steinwurf ApS
+// All Rights Reserved
+//
+// Signed-off-to-Steinwurf-by: Juan Cabrera <jcabre14@student.aau.dk>
+// Distributed under the "BSD License". See the accompanying LICENSE.rst file.
+
+
+#include <fstream>
+#include <cstdint>
+
+#include <tables/table.hpp>
+#include <tables/csv_format.hpp>
+#include <tables/json_format.hpp>
+#include <tables/python_format.hpp>
+
+/// @example creating_printing_table.cpp
+///
+/// Simple example showing how to create a table
+/// and print it in any desired format.
+
+int main()
+{
+
+    // In this example we will generate 100 random numbers. We then
+    // will store those numbers in a table with two columns:
+    // The random number and the iteration in wich it was generated.
+
+    // We set the names for the ouput files.
+    std::string out_csv = "out.csv";
+    std::string out_json = "out.json";
+    std::string out_python = "out_python.json";
+
+    // We will make the table object where the
+    // results are going to be stored.
+    tables::table results;
+
+    // We first create the two columns of the table.
+    if (!results.has_column("Iteration"))
+        results.add_column("Iteration");
+
+    if (!results.has_column("Number"))
+        results.add_column("Number");
+
+    for (uint8_t i = 0; i < 100; ++i)
+    {
+        // Generate the random number.
+        uint32_t random_number = rand();
+
+        // Add a row to the table.
+        results.add_row();
+
+        // Store the generated number and the iteration
+        // in the table.
+        results.set_value("Iteration", i);
+        results.set_value("Number", random_number);
```

Comment:
Maybe we could show that you can store other types as well here.
Something like this could work:
```
results.set_value("Classification", random_number > RAND_MAX / 2 ? "low" : "high");
```

You will of course need to add the column and mention it in the documentation.